### PR TITLE
Create Remove-DbaDbBackupRestoreHistory

### PR DESCRIPTION
**Pull request by ijeb**
## Type of Change

 - [ ] Bug fix (non-breaking change, fixes #<enter issue number>)
 - [x] New feature (non-breaking change, adds functionality) #3873
 - [ ] Breaking change (effects multiple commands or functionality)
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] Nunit test is included
 - [ ] Documentation
 - [ ] Build system
 

### Purpose
Implement #3873

### CODE REVIEW

**Comment by niphlod** [comment]
a .ps1 is missing here ?! ;-)

File: functions/Remove-DbaDbBackupRestoreHistory.ps1 [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        [DbaInstanceParameter[]]$SqlInstance,
+        [parameter(Mandatory = $false)]
+        [Alias("Credential")]
+        [PSCredential]
+        $SqlCredential,
+        [parameter(Mandatory = $false, ParameterSetName = "instance")]
+        $KeepDays = 30,
+        [parameter(Mandatory = $false, ParameterSetName = "instancedatabase")]
+        [Alias("Databases")]
+        [object[]]$Database,
```
Comment (potatoqualitee) [inline functions/Remove-DbaDbBackupRestoreHistory.ps1]: Everything looks great! The only thing I'd change is minor. We used to do `[object[]]$Database,` but have switched to `[string[]]$Database,`

File: functions/Remove-DbaDbBackupRestoreHistory.ps1 [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        [Parameter(ValueFromPipeline, Mandatory, ParameterSetName = "database")]
+        [Microsoft.SqlServer.Management.Smo.Database[]]$InputObject,
+        [Alias('Silent')]
```
Comment (potatoqualitee) [inline functions/Remove-DbaDbBackupRestoreHistory.ps1]: Oh, also new commands dont need this alias

**Comment by potatoqualitee** [comment]
I'll update post merge, ty!

**Comment by potatoqualitee** [comment]
Sorry got too excited, I do need to make more changes, I'll ping you on the PR

**Pull request merged**